Presentation.Web/Content/bootstrap-responsive.css

```diff
@@ -0,0 +1,1092 @@
+/*!
+ * Bootstrap Responsive v2.2.2
+ *
+ * Copyright 2012 Twitter, Inc
+ * Licensed under the Apache License v2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
```

Comment:
Du har en masses added css og pnger her under content, så dit PR er ikke clean endnu :-)

Reply:
Fjerner filerne (/Images og bootstrap-*) - må have været snavs i rørene .. den eneste pakke jeg har eksperimenteret med er SwashBucle.Examples (for at få SwaggerResponseHeader)